sdmetrics/single_table/bayesian_network.py

```diff
@@ -16,14 +18,6 @@ class BNLikelihoodBase(SingleTableMetric):
 
     @classmethod
     def _likelihoods(cls, real_data, synthetic_data, metadata=None, structure=None):
-        try:
-            from pomegranate import BayesianNetwork
-        except ImportError:
-            raise ImportError(
-                'Please install pomegranate with `pip install sdmetrics[pomegranate]`. '
-                'Python 3.13 is not supported.'
```

Comment:
I think we still want this import error but we can drop the Python 3.13 sentence